lib/yabeda/sidekiq.rb

```diff
@@ -59,6 +60,16 @@ module Sidekiq
           sidekiq_queue_latency.set({ queue: queue.name }, queue.latency)
         end
 
+        now = Time.zone.now
+        ::Sidekiq::Workers.new.each do |process, thread, msg|
+          payload = msg['payload']
+
+          sidekiq_worker_runtime.set(
+            {queue: payload['queue'], worker: payload['class']},
+            now - Time.at(msg['run_at'])
+          )
+        end
```

Comment:
Another problem here is that when processes are idle, `Sidekiq::Workers.new.each` returns zero entries. That means that in monitoring you will continue to see last recorded values. If last finished job was long-running one, your alert won't be resolved.

Not sure how it is better to solve it. One option is to remember all ever seen labels and forcibly set those that was not seen now to empty value? :thinking: